concent-builder/files/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,76 @@
+properties([[$class:
+'GithubProjectProperty',
+    displayName: '',
+    projectUrlStr: 'https://github.com/bartoszbetka/concent-deployment.git/'],
+pipelineTriggers([githubPush()])])
+
+stage ('Checkout repository') {
+    dir('/home/jenkins/concent-deployment/') {
+        git url:
+        'https://github.com/bartoszbetka/concent-deployment.git',
```

Comment:
We already have the CI hook enabled in the repo at golemfactory, so I think this should now point at that repo rather than your fork.

Reply:
This change is in a CI branch, I keep old fork for tests